CONTRIBUTING.md

```diff
@@ -113,6 +111,89 @@ You can use the che command `dashboard_dev_server` (command pallette > Run task
 $ node_modules/.bin/gulp serve --server=<che_api_url>
 ```
 
+### Workspace Loader
+Workspace loader source code is located in [<this.repo>/workspace-loader](./workspace-loader/) folder.
+Here is the developer workflow if you want to contribute to it:
+
+#### Devfile for workspace loader development
+
+The devfile: [<this.repo>/workspace-loader/devfile.yaml](./workspace-loader/devfile.yaml)
+
+In this section, we show how to setup a Che 7 environment to work on the che workspace loader, and how to use it.
+For the whole workflows, we will need a workspace with such containers:
```

Comment:
If we actually need only one container, maybe rewrite this one item list to just a container... But up to you.

Reply:
fixed

Reply:
fixed